compiler/damlc/tests/bond-trading/Setup.daml

```diff
@@ -76,46 +81,46 @@ template HelperSetupJob
       controller party
       do create Helper with party
 
-marketSetupEntryExample party = do
-  party1 <- getParty party
+marketSetupEntryExample party = script do
+  party1 <- allocateParty party
   return MarketSetupEntry with
     party = party1
     bondEntries = [ BondEntry with isin = "1234"; amount = 100.0 ]
     cashEntries = [ CashEntry with currency = "USD"; amount = 100.0 ]
 
-marketSetupJobExample = do
-  acme <- getParty "Acme Bank"
+marketSetupJobExample acme = script do
   entries <- mapA marketSetupEntryExample ["party1", "party2", "party3", "party4", "party5"]
   return MarketSetupJob with
     issuer = acme
     entries
 
 
-setupExample = scenario do
-    passToDate $ date 2018 May 14
+setupExample = script do
+    setDate 2018 May 14
 
-    acme <- getParty "Acme Bank"
+    acme <- allocateParty "Acme Bank"
 
-    job <- marketSetupJobExample
-    submit acme do create job
+    job <- marketSetupJobExample acme
+    submit acme do createCmd job
 
     return ()
 
-setupExampleFull = scenario do
-    acme <- getParty "Acme Bank"
+setupExampleFull = script do
+    acme <- allocateParty "Acme Bank"
 
-    passToDate $ date 2018 May 14
+    setDate 2018 May 14
 
-    job <- marketSetupJobExample
-    jobCid <- submit acme do create job
+    job <- marketSetupJobExample acme
+    jobCid <- acme `submit` createCmd job
 
-    rs <- submit acme do exercise jobCid Process
+    rs <- acme `submit` exerciseCmd jobCid Process
 
     forA rs $ \r -> submit r.party do
```

Comment:
There's a chance for inline submit here too, but this line hasn't changed so maybe not

Reply:
Yeah this one I was hesitant about, given the complexity of the RHS